src/modules/application/components/HomePage/HomePage.jsx

```diff
@@ -1,7 +1,17 @@
 import React from "react";
 import PropTypes from "prop-types";
+import useFelaEnhanced from "hooks/useFelaEnhanced";
+import * as rules from "./HomePage.styles";
```

Comment:
Haha, do you remember our initial discussion at the meeting about the poll we had? It was about this. It is yet to be added to our VS extension, but this actually should be 
```
import * as felaRules from './HomePage.rules
```
The reason is `useFelaEnhanced` returns not only `styles` but `rules` and `theme` too and it would conflict with this import. And the reason to change the file name is that the file actually includes fela rules, so to follow the naming, we decided to change the file names.

Reply:
Haha, ok, renaming it in the next commit.